portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,31 +19,26 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  ArrayList<String> values = new ArrayList<String>(){
-    {
-        add("Nice");
-        add("Good");
-        add("Great");
-    }
-  };
+  ArrayList<String> values = new ArrayList<String>();
 
-  @Override
+  /*@Override
```

Comment:
i think you might want to keep this function around instead of getting rid of it. one common design pattern for forms is:

user loads a page with the form (get request)
user fills in form and clicks button (post request)
server processes that data and redirects back to the original page (the tutorial suggests using response.sendRedirect())
user's browser loads the page again (get request) and now it shows the data

so if you follow this pattern, you will need both the doGet and doPost methods

Reply:
Yes! Makes sense.